### PR TITLE
updated to work with foreman-csv plugin

### PR DESCRIPTION
**Pull request by thomasmckay**
fixed roles

### CODE REVIEW

**Comment by thomasmckay** [comment]
doc updates in another PR when time allows

File: lib/hammer_cli_csv/roles.rb [hunk 1]
```diff
@@ -84,9 +84,7 @@ def create_roles_from_csv(line)
             @existing_roles[name] = role['id']
           else
             print "Updating role '#{name}'..." if option_verbose?
-            @api.resource(:roles).call(:update, {
-                'id' => @existing_roles[name]
-            })
+            # Nothing to update
```
Comment (komidore64) [inline lib/hammer_cli_csv/roles.rb]: should you print out that you're updating a role if you actually aren't?

Comment (thomasmckay) [inline lib/hammer_cli_csv/roles.rb]: Actually, in csv I merge the concept of "role" and "filter" into the higher level concept of "role." Filters, in Foreman, are never re-used between the role models. So each line of CSV modifies a role w/ filters but the role itself only has a "name" attribute which I don't allow changing. Hence, nothing to update in that section of code.

Comment (komidore64) [inline lib/hammer_cli_csv/roles.rb]: i'm referring to the line above this. `print "Updating role ..."` Why are you printing that if you aren't updating the role?

Comment (komidore64) [inline lib/hammer_cli_csv/roles.rb]: okay, explained in IRC

**Comment by komidore64** [comment]
ack

**Pull request merged**